python/cudf_polars/cudf_polars/dsl/expressions/binaryop.py

```diff
@@ -42,6 +42,7 @@ def __init__(
             op = BinOp._BOOL_KLEENE_MAPPING.get(op, op)
         self.op = op
         self.children = (left, right)
+        self.is_pointwise = all(c.is_pointwise for c in self.children)
```

Comment:
*suggestion*: I think that `is_pointwise` should be one-level, and it should be the responsibility of the evaluator to care about whether the children are pointwise.

So for `BinOps` `self.is_pointwise = True`.